servers/quarkus-server/src/main/java/org/projectnessie/server/configchecks/ConfigChecks.java

```diff
@@ -36,6 +37,11 @@ public class ConfigChecks {
   @Inject QuarkusJdbcConfig jdbcConfig;
 
   public void configCheck(@Observes StartupEvent event) {
+    if (System.getProperty("org.gradle.test.worker") != null) {
```

Comment:
Isn't it preferable to reduce the log threshold for this class in tests?

Reply:
yea